### DISCUSSION TITLE
How would you be called/recognized in KodaDot waters?

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#1927] [General]
We are cracking on the way how you would like to be called on chat, meta-hours, notes whats over as we don't want to call someone by surname but rather the first name which you prefer and sounds more unique or better for one's ear. Also calling by handle works as well.

For example, most people call me Matej or wao, hence I have handle yangwao.

I guess great example is @kkukelka we've started calling Kilian, @roiLeo it's better called Leo and with @prachi00 would be better Darshana &c, feel free raise objection, suggestion what's over! :)

**Comment by prachi00** [comment]
For me, anything works Darshana is my first name and Prachi is my short name that my friends and family use 😊
